src/main/java/core/utils/EncryptorAES.java

```diff
@@ -0,0 +1,110 @@
+     * @return Información encriptada.
+     * @throws UnsupportedEncodingException
+     * @throws NoSuchAlgorithmException
+     * @throws InvalidKeyException
+     * @throws NoSuchPaddingException
+     * @throws IllegalBlockSizeException
+     * @throws BadPaddingException
+     */
+    public static String encryptText(final String text, final String incomingSecretKey)
+            throws UnsupportedEncodingException, NoSuchAlgorithmException, InvalidKeyException, NoSuchPaddingException,
+            IllegalBlockSizeException, BadPaddingException {
+        SecretKeySpec secretKey = createKey(incomingSecretKey);
+        Cipher cipher = Cipher.getInstance("AES/ECB/PKCS5Padding");
+        cipher.init(Cipher.ENCRYPT_MODE, secretKey);
+        byte[] encryptedData = text.getBytes("UTF-8");
+        byte[] encryptedBytes = cipher.doFinal(encryptedData);
+        String encryptedText = Base64.getEncoder().encodeToString(encryptedBytes);
+        return encryptedText;
+    }
+
+    /**
+     * Decrypts the given encrypted text with the encryption key.
+     *
+     * @param encryptedData     the encrypted data.
+     * @param incomingSecretKey encryption key.
+     * @return decrypted text.
+     * @throws UnsupportedEncodingException
+     * @throws NoSuchAlgorithmException
+     * @throws InvalidKeyException
+     * @throws NoSuchPaddingException
+     * @throws IllegalBlockSizeException
+     * @throws BadPaddingException
+     */
+    public static String decryptText(final String encryptedData, final String incomingSecretKey)
+            throws UnsupportedEncodingException, NoSuchAlgorithmException, InvalidKeyException, NoSuchPaddingException,
+            IllegalBlockSizeException, BadPaddingException {
+        SecretKeySpec secretKey = createKey(incomingSecretKey);
+        Cipher cipher = Cipher.getInstance("AES/ECB/PKCS5PADDING");
+        cipher.init(Cipher.DECRYPT_MODE, secretKey);
+        byte[] encryptedBytes = Base64.getDecoder().decode(encryptedData);
+        byte[] decryptedData = cipher.doFinal(encryptedBytes);
+        String data = new String(decryptedData);
+        return data;
+    }
+
+    /**
+     * Gets the decrypted value.
+     *
+     * @param incomingText text to decrypt.
+     * @param incomingKey  key to decrypt.
+     * @return a string with the decrypted text.
+     * @throws Exception
+     */
+    public static String getDecryptedValue(final String incomingText, final String incomingKey) throws Exception {
+        String simpleText = "";
+        try {
+            final String key = incomingKey;
+            String text = incomingText;
+            EncryptorAES encryptor = new EncryptorAES();
+            simpleText = encryptor.decryptText(text, key);
+        } catch (UnsupportedEncodingException | NoSuchAlgorithmException | InvalidKeyException | NoSuchPaddingException
+                | IllegalBlockSizeException | BadPaddingException ex) {
+            System.out.println(ex);
```

Comment:
you should throw an exception. Instead of just printing it.

Reply:
Already done.